README.md

```diff
@@ -81,4 +82,59 @@ look like this:
     |-- <QBIC sample code>.fastq.gz.sha256sum
 ```
 
+### NGS single-end / paired-end data with metadata (deprecated)
 
+**Disclaimer!**  
+This data format is targeted for a single use case and should not be
+used for general data registration purposes. Please use the
+[NGS single-end / paired-end data](#ngs-single-end--paired-end-data)
+format for now.
+
+**Responsible dropbox:**
+[QBiC-register-imgag-dropbox](drop-boxes/register-imgag-dropbox)
+
+**Resulting data model in openBIS**  
+Q_TEST_SAMPLE -> Q_NGS_SINGLE_SAMPLE_RUN (with sample code) -> DataSet
+of type Q_NGS_RAW_DATA (directory with raw sequencing files contained)
+
+Example sample ids:
+
+QABCD001AE (Analyte, Q_TEST_SAMPLE)  
+NGS[0-9]{2}QABCS001AE (Sequencing Result, Q_NGS_SINGLE_SAMPLE_RUN) where
```

Comment:
Since we also give example for the codes I think its ok to also avoid the regex here
```suggestion
NGS23QABCS001AE (Sequencing Result, Q_NGS_SINGLE_SAMPLE_RUN) where
```

Reply:
I think it is important to now, how the code will look like. It will be always to digit, from 00-99.